backend/graphql/types/userType.ts

```diff
@@ -16,6 +16,16 @@ const userType = gql`
     currentlyTeachingJM: Boolean
   }
 
+  type TeacherDTO {
+    id: ID!
+    firstName: String!
+    lastName: String!
+    email: String!
+    grades: [String]
```

Comment:
```suggestion
    grades: [String!]!
    currentlyTeachingJM: Boolean!
```
i think we can add this as required, given it is a teacher